### DISCUSSION TITLE
Uncaught TypeError: name is null

### DISCUSSION THREAD

**Discussion by bobimicroweber** [livewire/livewire#4890] [1. Help]
Uncaught TypeError: name is null
    setAttribute index.js:151
    startLoading LoadingStates.js:222
    doAndSetCallbackOnElToUndo LoadingStates.js:281
    startLoading LoadingStates.js:220
    startLoading LoadingStates.js:208
    setLoading LoadingStates.js:179
    LoadingStates LoadingStates.js:68
    value MessageBus.js:17
    value MessageBus.js:16
    call HookManager.js:38
    callHook Store.js:121
    sendMessage index.js:244
    value index.js:252
    later debounce.js:8
    setTimeout handler*debounce/< debounce.js:12
    value index.js:225
    debouncedHandler node_initializer.js:243
    value index.js:582
    debouncedHandler node_initializer.js:201
    attachListener node_initializer.js:259
    attachDomListener node_initializer.js:170
    initialize node_initializer.js:42
    initialize node_initializer.js:15
    onNodeAdded index.js:492
    callHook morphdom.js:35
    handleNodeAdded morphdom.js:140
    handleNodeAdded morphdom.js:158
    handleNodeAdded morphdom.js:162
    handleNodeAdded morphdom.js:162
    handleNodeAdded morphdom.js:162
    handleNodeAdded morphdom.js:162
    morphEl morphdom.js:407
    morphEl morphdom.js:219
    morphEl morphdom.js:332
    morphEl morphdom.js:219
    morphEl morphdom.js:332
    morphEl morphdom.js:219
    morphEl morphdom.js:332
    morphEl morphdom.js:219
    morphEl morphdom.js:332
    morphEl morphdom.js:219
    morphEl morphdom.js:332
    morphEl morphdom.js:219
    morphEl morphdom.js:332
    morphEl morphdom.js:219
    morphEl morphdom.js:332
    morphEl morphdom.js:219
    morphEl morphdom.js:332

**Comment by joshhanley** [comment]
@bobimicroweber please share a description of your issue (not just a stack trace), as well as any relevant code, so we my assist.

Thanks!
